MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -253,11 +230,8 @@
     private int astechPoolOvertime;
     private int medicPool;
 
-    private int lastTeamId;
-    private int lastPartId;
-    private int lastForceId;
-    private int lastMissionId;
-    private int lastScenarioId;
+    // indicates whether or not the campaign should be gzipped, if possible.
+    private Boolean preferGzippedOutput;
```

Comment:
This is not initialized until you load a campaign file. If you start a new campaign it has the value of null and throws an NPE when you attempt to save it. I don't see a reason to use an Object wrapper here and would use a simple primitive `boolean` instead.

Reply:
My editor appears to prefer `Boolean` over `boolean`, don't know why. Fixed.